apps/server/src/institutions/institutionResolver.ts

```
@@ -35,72 +36,93 @@ const getAggregatorByVolume = (
   })?.[0] as Aggregator;
 };
 
-export async function resolveInstitutionAggregator(
-  institutionId: string,
-  jobTypes: ComboJobTypes[],
-): Promise<ResolvedInstitution> {
-  const institution = await getInstitution(institutionId);
-  const preferences = await getPreferences();
-  const aggregatorsWithAtLeastPartialSupport: Aggregator[] =
-    getAvailableAggregators({
+export async function resolveInstitutionAggregator({
+  aggregatorOverride,
+  jobTypes,
+  ucpInstitutionId,
+}: {
+  aggregatorOverride?: Aggregator;
+  jobTypes: ComboJobTypes[];
```

jobTypes doesn't seem to be used if aggregatorOverride is passed in, should this param be optional too?

---

I think I'd prefer to keep it required, because it will fail if it's not provided for the non aggregator override case. The function runs fine regardless of whether aggregatorOverride is passed